### PR TITLE
Homepage redesign

### PR DESCRIPTION
**Pull request by RnbWd**

### CODE REVIEW

File: BrightID/src/components/NewConnectionsScreens/MyCodeScreen.js [hunk 1]
```diff
@@ -29,103 +28,65 @@ import { startConnecting, stopConnecting } from './actions/connecting';
...
-      : undefined;
-    const prevQrString = prevProps.myQrData
-      ? prevProps.myQrData.qrString
-      : undefined;
-    if (currentQrString !== prevQrString) {
-      this.checkQrCode();
+  const timerTick = () => {
+    let countdown = ttl - (Date.now() - timestamp);
+    if (countdown <= 0) {
+      dispatch(startConnecting());
```
Comment (TripleSpeeder) [inline BrightID/src/components/NewConnectionsScreens/MyCodeScreen.js:55]: I think this is not necessary, as checkQrCode() whithin the useEffect block below will already dispatch startConnecting() as soon as the qrString from redux is not set.

File: BrightID/src/components/HomeScreen.js [hunk 2]
```diff
@@ -10,284 +9,333 @@ import {
...
-    isEditing: false,
-    name: '',
-  };
+  const verified = useMemo(() => verifications.includes('BrightID'), [
+    verifications,
+  ]);
 
-  componentDidMount() {
-    const { navigation, dispatch, photo, name } = this.props;
+  useEffect(() => {
```
Comment (TripleSpeeder) [inline BrightID/src/components/HomeScreen.js:57]: You could refactor this to make use of the react-navigation hooks. (https://reactnavigation.org/docs/use-focus-effect)

File: BrightID/src/components/NewConnectionsScreens/ScanCodeScreen.js [hunk 3]
```diff
@@ -32,214 +43,219 @@ function validQrString(qrString) {
...
-          <View style={styles.scanTextContainer}>
-            <TextInput
-              ref={(c) => {
-                this.textInput = c;
+        <View style={styles.cameraContainer}>
+          {!scanned ? (
+            <RNCamera
+              style={styles.cameraPreview}
+              captureAudio={false}
+              onBarCodeRead={handleBarCodeRead}
```
[the remark this answers is not in the document]
Comment (RnbWd) [reply BrightID/src/components/NewConnectionsScreens/ScanCodeScreen.js:139]: I was actually going to add a deep link, it's still possible to test with this, right?

[the remark this answers is not in the document]
Comment (RnbWd) [reply BrightID/src/components/NewConnectionsScreens/ScanCodeScreen.js]: Do you still think we should add this?

File: BrightID/src/components/HomeScreen.js (hunk 2 above)
Comment (RnbWd) [reply BrightID/src/components/HomeScreen.js:51]: thanks, I'm using that effect on a different page, and should probably use it here as well

File: BrightID/src/components/NewConnectionsScreens/ScanCodeScreen.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (TripleSpeeder) [reply BrightID/src/components/NewConnectionsScreens/ScanCodeScreen.js]: Not necessary, with deep-linking working I can also use the terminal.

**Pull request merged**